### DISCUSSION TITLE
graphql/codegen/docs/advanced/programmatic-usage

### DISCUSSION THREAD

**Discussion by giscus** [dotansimha/graphql-code-generator#8897] [Docs Discussions]
# graphql/codegen/docs/advanced/programmatic-usage

GraphQL Code Generator Documentation

https://the-guild.dev/graphql/codegen/docs/advanced/programmatic-usage

**Comment by paul-vd** [comment]
> We are using this API in the live demo on the GraphQL Code Generator website. [The code is here](https://github.com/dotansimha/graphql-code-generator/blob/master/website/src/components/live-demo/generate.js).

The link is missing: https://github.com/dotansimha/graphql-code-generator/blob/master/website/src/components/live-demo/generate.js

**Reply by saihaj** [reply]
Thanks fixed it https://github.com/dotansimha/graphql-code-generator/commit/6ff5f0582a31485ce4399ea55dfc5b3b5b2f6731
